### ISSUE TITLE
Cannot debug the test

### ISSUE THREAD

**Issue by TusharJadhav** [Microsoft/TestAdapterForGoogleTest#2]
I am not able to debug the google test using this extension. When I right click and select "Debug Test" for the test that I want to debug, I get the below error -

ERROR: Failed to run test executable 'D:\my\bin\SystemTest.exe': Could not load file or assembly 'Microsoft.VisualStudio.Shell.15.0, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
ERROR: Stacktrace:
   at GoogleTestAdapter.TestAdapter.Framework.VsDebuggerAttacher.AttachDebugger(Int32 processId)
   at GoogleTestAdapter.Helpers.ProcessExecutor.NativeMethods.ExecuteCommandBlocking(String command, String parameters, String workingDir, String pathExtension, IDebuggerAttacher debuggerAttacher, Action`1 reportOutputLine, Action`1 reportProcessId)
   at GoogleTestAdapter.Helpers.ProcessExecutor.ExecuteCommandBlocking(String command, String parameters, String workingDir, String pathExtension, Action`1 reportOutputLine)
   at GoogleTestAdapter.Runners.SequentialTestRunner.RunTestExecutableWithNewFramework(String executable, String workingDir, Args arguments, IProcessExecutor executor, StreamingStandardOutputTestResultParser streamingParser)
   at GoogleTestAdapter.Runners.SequentialTestRunner.TryRunTests(String executable, String workingDir, String baseDir, Boolean isBeingDebugged, IDebuggedProcessLauncher debuggedLauncher, Args arguments, String resultXmlFile, IProcessExecutor executor, StreamingStandardOutputTestResultParser streamingParser)
   at GoogleTestAdapter.Runners.SequentialTestRunner.RunTests(String executable, String workingDir, String baseDir, Boolean isBeingDebugged, IDebuggedProcessLauncher debuggedLauncher, Args arguments, String resultXmlFile, IProcessExecutor executor, StreamingStandardOutputTestResultParser streamingParser)

**Comment by LukaszMendakiewicz** [comment]
Thanks for the report. What is your Visual Studio version (in Help > About)?

**Comment by TusharJadhav** [comment]
I have Visual Studio 2015 and 2017 installed side by side. But I am trying to debug my tests in VS2017. The versions are -
VS2017 - Version 15.2 (26430.13) Release. Microsoft .NET Framework Version 4.6.01038
VS2015 - Version 14.0.25431.01 Update 3 Release. Microsoft .NET Framework Version 4.6.01038

I tried debugging the solution as well and noticied that the AssemblyResolve Event in \TestAdapterForGoogleTest\GoogleTestAdapter\TestAdapter\Framework\VsDebuggerAttacher.cs is called for loading Microsoft.VisualStudio.Shell.15.0 since it is not resolved this method is throwing the exception that I pasted above.

**Comment by LukaszMendakiewicz** [comment]
Did you switch "Use new test execution framework (experimental)" in Options to True? If so, please switch it back to False as a workaround.
We expect the fix for this to be included in the next update to the extension.

**Comment by TusharJadhav** [comment]
I have modified the GTA source for my needs and have made changes only to the new test execution framework. I will have to do more work to support my changes to the old execution framework. What's the ETA for the next update to the extension?
In the mean time, I cloned the Christian Soltenborn original source from which the Microsoft repo is forked and applied my changes which seem to be working. So for the time being I am using his GTA extension. Not exactly sure what are the advantages/ disadvantages of doing so.

**Comment by LukaszMendakiewicz** [comment]
We are collaborating with Christian on developing the extensions. GTA (Christian's repo) is to be the "bleeding edge" version, TAfGT (Microsoft's repo) is to be a stable version trailing behind GTA. Currently we are still in the process of setting everything up, so there are some differences between the repositories - most notably TAfGT offers project and item templates, as well as updated icons matching VS style better. These should go to GTA soon when Christian merges our changes back in. On TAfGT side, I have merged GTA's version 0.10.1 in last week (we were based on 0.9.1 before) and we will be releasing an  update to Visual Studio Marketplace _really soon_.
You are most welcome to contribute your changes to either repository.

**Comment by LukaszMendakiewicz** [comment]
@TusharJadhav we have now release the updated version (0.10.1) of [our extension](https://marketplace.visualstudio.com/items?itemName=VisualCPPTeam.TestAdapterforGoogleTest).

I hope that between this release and the custom changes you have made yourself, the issue is now resolved so we can close it here?

**Comment by LukaszMendakiewicz** [comment]
I'm closing this issue due to inactivity. Please report if this is still an issue.

**Comment by TusharJadhav** [comment]
Sorry for late response since I got busy with another project. I just tried downloading the latest source but it fails to even compile. Below are the compiler errors 

Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not copy the file "D:\Development\TestAdapterForGoogleTest\out\binaries\GoogleTestAdapter\Debug\Packaging.GTA\\..\VsPackage.GTA\GoogleTestAdapter.VsPackage.GTA.pkgdef" because it was not found.	Packaging.GTA	D:\Development\TestAdapterForGoogleTest\GoogleTestAdapter\Packaging.GTA\Packaging.GTA.csproj	115	

Error		An exception was thrown while trying to compile the transformation code. The following Exception was thrown:
System.IO.FileNotFoundException: Could not find file 'D:\Development\TestAdapterForGoogleTest\GoogleTestAdapter\VsPackage.Tests.Generated\GoogleTestAdapter.Tests.Common.dll'.
File name: 'D:\Development\TestAdapterForGoogleTest\GoogleTestAdapter\VsPackage.Tests.Generated\GoogleTestAdapter.Tests.Common.dll'
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Roslyn.Utilities.FileUtilities.OpenFileStream(String path)
   at Microsoft.CodeAnalysis.MetadataReference.CreateFromFile(String path, MetadataReferenceProperties properties, DocumentationProvider documentation)
   at Microsoft.VisualStudio.TextTemplating.CompilerBridge.<>c.<.ctor>b__15_0(String x)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Linq.Enumerable.<UnionIterator>d__66`1.MoveNext()
   at System.Linq.Enumerable.<UnionIterator>d__66`1.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at System.Collections.Immutable.ImmutableArray.CreateRange[T](IEnumerable`1 items)
   at Microsoft.CodeAnalysis.ImmutableArrayExtensions.AsImmutableOrEmpty[T](IEnumerable`1 items)
   at Microsoft.CodeAnalysis.Compilation.ValidateReferences[T](IEnumerable`1 references)
   at Microsoft.CodeAnalysis.CSharp.CSharpCompilation.WithReferences(IEnumerable`1 references)
   at Microsoft.CodeAnalysis.CSharp.CSharpCompilation.CommonWithReferences(IEnumerable`1 newReferences)
   at Microsoft.VisualStudio.TextTemplating.CompilerBridge.PrepareNewCompilation()
   at Microsoft.VisualStudio.TextTemplating.CompilerBridge.Compile()
   at Microsoft.VisualStudio.TextTemplating.TransformationRunner.Compile(String source, String inputFile, IEnumerable`1 references, Boolean debug, SupportedLanguage language, String compilerOptions). Line=0, Column=0	VsPackage.Tests.Generated
